lib/sensu/extensions/occurrences.rb

```diff
@@ -22,8 +22,11 @@ def description
       def event_filtered?(event)
         check = event[:check]
         occurrences = check[:occurrences] || 1
+        watermark = event[:occurrences_watermark] || 1
```

Comment:
I think I added the default to make things work before I updated the event spec. Although gem extension loading is new in 0.26, I figured there's an outside chance that someone will try to manually install this filter extension in their extensions directory on an older version.

I'll remove the default if you prefer.